Garrett-Iannuzzi/hot_like_sauce
# Header setup and style

## What does this PR do?
- Adds icons for the menu, cart, and logo
- positions the header 

## How to test?
- pull down `header-setup`
- run `npm start` and see changes

## Issues:
- this PR closes #13 

## Notes:
- None
